### PR TITLE
fix(StyledImageWithFallback loading): handle slow loading and error

### PR DESCRIPTION
**Pull request by XAndreJentzsch**
# Description
 
This PR adds an additional loading state to the StyledImageWithFallback. 
While loading the image now the skeleton animation is visible.

Fixes # (Issue)

## Type of change

Please delete options that are not relevant.

-   [x] Minor change (non-breaking change, e.g. documentation adaption)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)

# Checklist:

-   [x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [ ] I have added tests that prove my fix or my feature works
-   [ ] New and existing tests pass locally with my changes
-   [x] My changes contain no console logs

### CODE REVIEW

**Review by GailMelanie — APPROVED** [review]

File: src/app/[locale]/list/_components/StyledImageWithFallBack.tsx [hunk 1]
```diff
@@ -23,23 +23,36 @@ type StyledImageWithFallBackProps = {
 export const ImageWithFallback = (props: StyledImageWithFallBackProps) => {
     const { src, alt, size, onClickHandler } = props;
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/StyledImageWithFallBack.tsx:24]:
```suggestion
export const ImageWithFallback = ({ src, alt, size, onClickHandler }: StyledImageWithFallBackProps) => {
```

File: src/app/[locale]/list/_components/StyledImageWithFallBack.tsx [hunk 2]
```diff
@@ -49,21 +62,20 @@ export const ImageWithFallback = (props: StyledImageWithFallBackProps) => {
...
+            onClick={() => onClickHandler?.call(this)}
         />
     );
 
     return (
         <>
-            {(!hasError && props.src) ? (
-                ImageContent
+            {(!hasError && src) ? (
+                <div style={{ position: 'relative', height: size, width: '100%', maxWidth: size }}>
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/StyledImageWithFallBack.tsx:75]: why div and not Box?

**Review by JonathanXITASO — APPROVED** [review]

File: src/app/[locale]/list/_components/StyledImageWithFallBack.tsx (hunk 1 above)
Comment (XAndreJentzsch) [reply src/app/[locale]/list/_components/StyledImageWithFallBack.tsx]: Better this way, thanks :)

File: src/app/[locale]/list/_components/StyledImageWithFallBack.tsx (hunk 2 above)
Comment (XAndreJentzsch) [reply src/app/[locale]/list/_components/StyledImageWithFallBack.tsx]: Changed to Box

**Pull request merged**